index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <title>Web Developer Test</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+
+<body class="whiteFont">
+    <div class="bordercontainer">
+        <header>
+            <h1 class="heading">CYF</h1>
+        </header>
+        <nav-bar>
+            <button>Menu 1</button>
+            <button>Menu 2</button>
+            <button>Menu 3</button>
+        </nav-bar>
+        <section>
+            <h2 class="aboutCYF red">About CYF</h2>
+        </section>
+        <article>
+            <h3>What We Do</h3>
+            <p>
+                <strong class="biggertext">CYF</strong> is a fictitious company-lorem ipsum dolor sit amet, whatever. CYF
```

Comment:
class names, either use dashes as separators which is preferred (so bigger-text) or camelCase. The important thing is to be consistent, and not have it all lowercase as it is hard to read